File: Basic/Calculate Factorial of A Number/SolutionByGurirath.cpp
#include <iostream>
using namespace std;

int fact(int num)
{
    if (num >= 1)
        return num * fact(num - 1);
    else
        return 1;
}

int main()
{
    int num;
    cout << "enter number";
    cin >> num;
    cout << "fact of " << num << "\t:\t" << fact(num) << "\num";
    return 0;
}

File: Basic/Calculate Factorial of A Number/SolutionByGurirath.java

/**
 Write a program to print the factorial of a number.
 */
import java.util.*;
public class SolutionByGurirath
{
    public static void main(String args[])
    {
        Scanner kb= new Scanner(System.in);
        int f=1,i;
        System.out.println("Enter a number to print its factorial");
        int n=kb.nextInt();
        for(i=1;i<=n;i++)
        {
            f=f*i;
        }
        System.out.println("Factorial:"+f);
        
        
    }
}
